docs/dev/error-messages.md

```diff
@@ -42,7 +42,7 @@ The expected rate of occurrence for this error is very low over a sustained peri
 
 **Cause(s)**
 
-The first explanation is that the VM crashed. When that happens, the VM stops communicating with our OnDemand services and the VM is effectively "lost." This can happen when:
+The first explanation is that the VM crashed. When that happens, the VM stops communicating with our OnDemand services and the VM is effectively "lost". This can happen when:
```

Comment:
I think this should be left as-is (period inside the quote), since we follow U.S. style

Reply:
Thanks for the correction!